src/en/news/blog/2021/pacific-usability-advanced-install/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Ceph Pacific Usability: Advanced Installation"
+date: "2021-08-23"
+author: "Paul Cuzner"
+---
+
+Starting with the Ceph Octopus release, Ceph provides its own configuration and management control plane in the form of the ‘mgr/orchestrator’ framework. This feature covers around 90% of the configuration and management requirements for Ceph. Why not 100%...well, there are some tasks like preparing hosts and purging a cluster that need to be performed from outside of the cluster! For those tasks take a look at [cephadm-ansible](https://github.com/ceph/cephadm-ansible/) and [ceph-salt](https://github.com/ceph/ceph-salt).
+
+Installation, on the other hand, is **completely covered** by the orchestrator!
+
+First a bit of background. The orchestrator/cephadm framework uses ssh under the covers. Typically, you don’t want the ssh user account used by the orchestrator module to be the root account - so you’ll need a user account set up on each node that has sudo privileges (just like in a Ansible controlled environment). This account must be able to issue commands and login, without a password. In this article, we'll be using an account called `cephadmin`, and will use the ssh keys on the bootstrap node’s cephadmin account for Ceph.
+
```

Comment:
s/a Ansible/an Ansible/ controlled environment.

Should the account name here be `cephadm` rather than `cephadmin`? (I know ceph-salt creates a `cephadm` account)

Reply:
the account can be whatever you want - it's up to the sysadmin